operations/patch_util.go

```diff
@@ -435,9 +430,16 @@ func gitLog(base, ref string) (string, error) {
 }
 
 func gitCommitMessages(base, ref string) (string, error) {
-	args := []string{"--no-show-signature", "--pretty=format:%B", fmt.Sprintf("%s@{upstream}..%s", base, ref)}
+	args := []string{"--no-show-signature", "--pretty=format:%B", "--reverse", fmt.Sprintf("%s@{upstream}..%s", base, ref)}
 	msg, err := gitCmd("log", args...)
-	return strings.Replace(msg, "\n\n", "\n", -1), err // remove multiple line format if applicable
+	if err != nil {
+		return "", errors.Wrap(err, "can't get messages")
+	}
+	// separate multiple commits with <-
+	msg = strings.Replace(msg, "\n\n", " <- ", -1)
```

Comment:
I worry that an actual commit message might have \n\n in it somehow, and then we'd be changing the actual message by adding this `<-` rather than just limiting space a bit. Or what if the msg ends with \n\n? Should we also trim " <- "?

Reply:
I updated the log format to `%s` which just prints the "subject" of the commit message, which is just the first line. If there are any newlines in the message they will be excluded.